.github/workflows/pkg.pr.new.yaml

```diff
@@ -0,0 +1,47 @@
+name: Publish Approved PR
+on:
+  pull_request_review:
+    types: [submitted]
```

Comment:
I think publishing this also can be trigger only when there's change in `packages`, similar to `build.yaml` workflow

```
pull_request_review:
    types: [submitted]
    paths:
      - 'packages/**'
```